src/tests/src/solver/optim-model-filler/test_componentFiller.cpp

```diff
@@ -738,4 +732,100 @@ BOOST_AUTO_TEST_CASE(one_var_with_param_objective)
     BOOST_CHECK_EQUAL(pb->getObjectiveCoefficient(pb->lookupVariable("componentA.x")), -25);
 }
 
+BOOST_AUTO_TEST_CASE(one_var_no_offset)
```

Comment:
This file contains already many tests and many lines of code.
As its name says, it's supposed to test component fillers. 
It seems we're doing more than that in this file.
Maybe we should start thinking about splitting this file.